Bottom: 2593107c4a09a6239078b8089af9ae634b2b5044
Top:    1b8319b68116f04133b63b912af3519f2e066eb8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 16:14:20 +0000

Refresh of use-calling-wrappers-rather

---

diff --git a/src/draw.c b/src/draw.c
index afad139..3083038 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1542,8 +1542,8 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.bgGC = hid_draw_make_gc (hid->graphics);
   Output.pmGC = hid_draw_make_gc (hid->graphics);
 
-  hid->graphics->set_color (Output.pmGC, "erase");
-  hid->graphics->set_color (Output.bgGC, "drill");
+  hid_draw_set_color (Output.pmGC, "erase");
+  hid_draw_set_color (Output.bgGC, "drill");
 
   if (item)
     {
